app/services/onlinetesting/phase1/Phase1TestEvaluation.scala

```diff
@@ -46,7 +52,7 @@ trait Phase1TestEvaluation {
         case (Green, Green) => Green
       }
 
-      SchemeEvaluationResult(scheme, result.toString)
+      SchemeEvaluationResult(schemeToEvaluate, result.toString)
```

Comment:
Though the change is not part of this PR ... any reason why we convert to result.toString, instead of using result which is type safe ??